src/lib.rs

```diff
@@ -69,10 +69,20 @@ that this copyright notice remain intact.
 //! let color_map = nq.color_map_rgba();
 //! ```
 
+// #![no_std]
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[macro_use]
+extern crate alloc;
+
+#[allow(unused_imports)]
+use num_traits::float::FloatCore;
```

Comment:
This would usually be a code smell. But since it is the entire point of the PR, what? How is it unused? At least it should have a `cfg` constraint to the relevant configurations.